### PR TITLE
docs: improve install.md

### PR DESCRIPTION
**Pull request by kenjis**
- fix Markdown format
- move the Migration step

### CODE REVIEW

**Review by datamweb — APPROVED** [review]
Suggestion, you can do it or pass it:

Codeigniter **v4.2.3 or later**

File: docs/install.md [hunk 1]
```diff
@@ -37,60 +40,36 @@ If you get the following error:
 
 1. Add the following to change your [minimum-stability](https://getcomposer.org/doc/articles/versions.md#minimum-stability) in your project `composer.json`:
 
-```
-    "minimum-stability": "dev",
-    "prefer-stable": true,
-```
+    ```
```
Comment (MGatner) [inline docs/install.md:43]: Is it worth highlighting partials?
```suggestion
    ```json
```

File: docs/install.md [hunk 2]
```diff
@@ -37,60 +40,36 @@ If you get the following error:
...
+    > composer require codeigniter4/shield:dev-develop
+    ```
 
-When you run `spark migrate --all`, if you get `Class "SQLite3" not found` error:
+    The above specifies `develop` branch.
+    See https://getcomposer.org/doc/articles/versions.md#branches
 
-1. Remove sample migration files in `tests/_support/Database/Migrations/`
-2. Or install `sqlite3` php extension
+    ```
```
Comment (MGatner) [inline docs/install.md:57]:
```suggestion
    ```shell
```

File: docs/install.md [hunk 3]
```diff
@@ -37,60 +40,36 @@ If you get the following error:
...
 
 ### Command Setup
 
-1. Run the following command. This command handles steps 1-4 of *Manual Setup* and runs the migrations.
+1. Run the following command. This command handles steps 1-5 of *Manual Setup* and runs the migrations.
 
-```
-> php spark shield:setup
-```
+    ```
```
Comment (MGatner) [inline docs/install.md:70]:
```suggestion
    ```shell
```

File: docs/install.md [hunk 4]
```diff
@@ -99,43 +78,60 @@ your project.
...
+    public function initController(RequestInterface $request, ResponseInterface $response, LoggerInterface $logger)
+    {
+        $this->helpers = array_merge($this->helpers, ['setting']);
 
-    // Do Not Edit This Line
-    parent::initController($request, $response, $logger);
-}
-```
+        // Do Not Edit This Line
+        parent::initController($request, $response, $logger);
```
Comment (MGatner) [inline docs/install.md:104]: Side note... I've always hated that we have this there. No idea why there wasn't just a dedicated initialization function.

File: docs/install.md [hunk 5]
```diff
@@ -99,43 +78,60 @@ your project.
...
-```
+    ```php
+    service('auth')->routes($routes);
+    ```
 
 4. **Security Setup** Set `Config\Security::$csrfProtection` to `'session'` (or set `security.csrfProtection = session` in your `.env` file) for security reasons, if you use Session Authenticator.
 
+5. **Migration** Run the migrations.
+
+    ```
```
Comment (MGatner) [inline docs/install.md:120]:
```suggestion
    ```shell
```

**Review by MGatner — CHANGES_REQUESTED** [review]
Generally, what does indenting the triple-backtick blocks do? I thought those were supposed to be fully left-aligned, but it looks like they work either way.

File: docs/install.md (hunk 1 above)
Comment (kenjis) [reply docs/install.md]: Done.

File: docs/install.md (hunk 4 above)
Comment (kenjis) [reply docs/install.md:104]: What do you mean?
The Controller should have another initialization method?

Comment (MGatner) [reply docs/install.md:104]: Right. There's absolutely no benefit to having those parameters exposed and adding Aline that developers shouldn't touch. Call `initController()` (and make it `final`) and then add and call a blank `initialize()` method to `CodeIgniter\Controller` and extend that in `BaseController` instead. Bam: no more weird code in userland. 

Something for v5. 🤷‍♂️

**Review by MGatner — APPROVED** [review]

**Comment by MGatner** [comment]
Still curious if there is an advantage to indenting triple backticks. 👆

**Comment by kenjis** [comment]
Before:
Screenshot 2022-08-25 19 00 26

After:
Screenshot 2022-08-25 19 00 36

**Pull request merged**